packages/dashmate/docs/services/gateway.md

```
@@ -103,13 +103,13 @@ http2_protocol_options:
 
 ### Timeouts
 
-Different endpoints have different timeout requirements:
+Dashmate exposes per-endpoint timeout controls that are shared between rs-dapi and Envoy. Envoy is configured automatically with an additional five-second grace period so that rs-dapi can return structured errors before the proxy aborts a request.
 
 - Standard API endpoints: 10-15 seconds (fixed in Envoy configuration)
-- Core streaming endpoints: 600 seconds (10 minutes) (fixed in Envoy configuration)
-- waitForStateTransitionResult endpoint: 125 seconds (default)
-
-**Config option**: `platform.gateway.listeners.dapiAndDrive.waitForStResultTimeout`
+ - Core streaming endpoints: derived from `platform.dapi.rsDapi.timeouts.coreStreams` (default 600 000 ms)
```

markdownlint-cli2 (0.18.1)

Inconsistent indentation for list items at the same level
Expected: 0; Actual: 1

(MD005, list-indent)